packages/cashier-v2/src/lib/PaymentAgent/lib/PaymentAgentWithdrawal/components/PaymentAgentUnlistedWithdrawalForm/PaymentAgentUnlistedWithdrawalForm.tsx

```diff
@@ -0,0 +1,93 @@
+import React from 'react';
+import { Field, FieldProps, Form, Formik } from 'formik';
+import { useActiveAccount } from '@deriv/api-v2';
+import { LabelPairedCircleXmarkMdFillIcon, StandaloneArrowLeftBoldIcon } from '@deriv/quill-icons';
+import { Button, Input, Text } from '@deriv-com/ui';
+import styles from './PaymentAgentUnlistedWithdrawalForm.module.scss';
+
+const PaymentAgentUnlistedWithdrawalForm = () => {
+    const { data: activeAccount } = useActiveAccount();
+
+    return (
+        <div className={styles.container}>
+            <div className={styles['back-section']}>
+                <StandaloneArrowLeftBoldIcon data-testid='dt-back-arrow-icon' iconSize='md' />
+                <Text size='sm' weight='bold'>
+                    Back to list
+                </Text>
+            </div>
+            <Formik
+                initialValues={{
+                    accountNumber: '',
+                    amount: '',
+                }}
+                onSubmit={() => undefined}
```

Comment:
You can save the pointer, declaring a no-operation function outside your component and using it as via the reference in a handler

```
const noop = () => undefined 
```
```
onSubmit = {noop}
```

Reply:
sure @dan-deriv usually i do exactly the same, for now this is a kind of a plug and will be removed in the next PR, but let me fix it, thanks